subsys/bluetooth/host/gatt.c

```diff
@@ -275,6 +276,59 @@ static void sc_process(struct k_work *work)
 	atomic_set_bit(sc->flags, SC_INDICATE_PENDING);
 }
 
+#if defined(CONFIG_BT_SETTINGS_CCC_STORE_ON_WRITE)
+static struct gatt_ccc_store {
+	struct bt_conn *conn_list[CONFIG_BT_MAX_CONN];
+	struct k_delayed_work work;
+} gatt_ccc_store;
+
+static bool gatt_ccc_is_conn_queued(struct bt_conn *conn)
+{
+	if (conn == gatt_ccc_store.conn_list[bt_conn_get_id(conn)]) {
+		return true;
+	}
+
+	return false;
+}
```

Comment:
Minor, but this whole function could be just a single line:
```
return (conn == gatt_ccc_store.conn_list[bt_conn_get_id(conn)]);
```

Also, at least in the other similar boolean APIs I've tried to have the "is" right before the adjective, to make it flow like actual english language, so I'd prefer `gatt_ccc_conn_is_queued()`

Reply:
Done